src/app/ui/landing-page/landing-page.js

```diff
@@ -127,21 +123,14 @@ export class LandingPage extends BaseApi {
 
     let learnMore = () => { window.open('https://remix.readthedocs.io/en/latest/layout.html', '_blank') }
 
-    let closeAll = () => {
-      this.appStore.getActives()
-      .filter(({profile}) => !profile.required)
-      .forEach((profile) => { this.appManager.deactivateOne(profile.name) })
-    }
     let startSolidity = () => {
```

Comment:
I think it would be good practice to change all `var` and `let` into `const` if applicable.